### PR TITLE
config, env, folder structure setup for CC

### PR DESCRIPTION
**Pull request by sigadamvenkata**
config, env, folder structure setup for CC onboarding

### CODE REVIEW

File: tests/cc/febpub.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-restricted-syntax */
+import { fedpub } from '../../selectors/cc/fedpub.selectors.js';
+
+const { test } = require('@playwright/test');
+const fedpubSpec = require('../../features/cc/fedpub.sanity.js');
+
+const { features } = fedpubSpec;
+const { WebUtil } = require('../../libs/webutil.js');
+
+test('ulr has html ext' , async({page}) =>
```
Comment (skumar09) [inline tests/cc/febpub.test.js:10]: please pass `browser` object also to `test` method

Approving the PR, [ It's a initial PR to to onboard cc to Nala]

**Review by skumar09 — APPROVED** [review]

File: selectors/cc/febpub.selectors.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (JackySun9) [inline selectors/cc/febpub.selectors.js:1]: We are using Page Objects now, please check other code for sample

**Review by JackySun9 — CHANGES_REQUESTED** [review]

File: envs/envs.js [hunk 3]
```diff
@@ -15,4 +15,5 @@ module.exports = {
   '@blog_live': 'https://main--blog--adobe.hlx.live',
   '@local3000': 'http://localhost:3000',
   '@local6456': 'http://localhost:6456',
-};
+  '@cc_live' : 'https://main--cc--adobecom.hlx.live',
```
Comment (JackySun9) [inline envs/envs.js:18]: you don't have cc_prod here

**Review by JackySun9 — CHANGES_REQUESTED** [review]

File: selectors/cc/febpub.selectors.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+    '@gnav1item': '//div[@class="gnav-navitem"][1]',
+    '@gnavtrybtn': '//strong[@class="gnav-cta"]',
+       
+  };
+
```
Comment (chadsunvice) [inline selectors/cc/febpub.selectors.js:6]: For consistency all files end in newline, here there are trailing whitespaces.

File: features/cc/febpub.sanity.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    name: 'fedpub Component',
```
Comment (chadsunvice) [inline features/cc/febpub.sanity.js:2]: Keep uppercase for component name to keep consistency with others (eg: `FedPub Component`, or `Fedpub Component`), look cleaner in reports.

File: features/cc/febpub.sanity.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+      {
+        tcid : 1,
+        name: '@fedpub',
+        path: '/creativecloud/photography/hub/guides/camera-settings-for-rain-photography',
+        envs: '@cc_live @adobe_stage',
+        tags: '@cc @cc-fedpub',
+      },
+     ],
+  };
+
```
Comment (chadsunvice) [inline features/cc/febpub.sanity.js:13]: For consistency all files end in newline, here there are trailing whitespaces.

File: tests/cc/febpub.test.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+{await page.goto('https://www.stage.adobe.com/creativecloud/photography/hub/guides/camera-settings-for-rain-photography')} );
```
Comment (chadsunvice) [inline tests/cc/febpub.test.js:11]: We should avoid hardcoding URLs. Also format the code better. 

```
test('URL has html extension' , async({page}) => {
  await page.goto('https://www.stage.adobe.com/creativecloud/photography/hub/guides/camera-settings-for-rain-photography')};
  await expert(page).toHaveURL('html');
});
```

File: tests/cc/febpub.test.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+await expert(page).toHaveURL('html');
```
Comment (chadsunvice) [inline tests/cc/febpub.test.js:13]: For consistency all files end in newline, here there are trailing whitespaces.

**Review by chadsunvice — CHANGES_REQUESTED** [review]

File: selectors/cc/febpub.selectors.js (hunk 4 above)
Comment (sigadamvenkata) [reply selectors/cc/febpub.selectors.js]: removed the newline spaces

File: features/cc/febpub.sanity.js (hunk 6 above)
Comment (sigadamvenkata) [reply features/cc/febpub.sanity.js:13]: removed

File: tests/cc/febpub.test.js (hunk 7 above)
Comment (sigadamvenkata) [reply tests/cc/febpub.test.js]: removed

**Review by chadsunvice — APPROVED** [review]
Looks good!

**Review by JackySun9 — APPROVED** [review]

**Pull request merged**